src/mlpack/core/util/arma_traits.hpp

```diff
@@ -301,15 +311,23 @@ struct IsBaseMatType<arma::SpRow<eT>>
 template<typename T>
 struct IsArma
 {
-  constexpr static bool value = arma::is_arma_type<T>::value;
+  constexpr static bool value
+  =  arma::is_arma_type<T>::value
+  || arma::is_arma_cube_type<T>::value
+  || arma::is_arma_sparse_type<T>::value
+  ;
```

Comment:
```suggestion
  constexpr static bool value = arma::is_arma_type<T>::value ||
                                arma::is_arma_cube_type<T>::value ||
                                arma::is_arma_sparse_type<T>::value;
```
Style fix---this is more in line with the mlpack style guide.